app/models/pageflow/feed_entries_query.rb

```diff
@@ -0,0 +1,19 @@
+module Pageflow
+  class FeedEntriesQuery < ApplicationQuery
+    class Scope < Scope
+      attr_reader :scope, :page
+
+      def initialize(scope, page)
+        @scope = scope
+        @page = page
+      end
+
+      def resolve
+        scope \
+          .published_without_password_protection
+          .order(updated_at: :desc)
+          .page page
```

Comment:
ActiveAdmin [jumps though some hoops](https://github.com/activeadmin/activeadmin/blob/6a0f2481761ee1387b30b9c89353faa4b4ffb359/lib/active_admin/resource_controller/data_access.rb#L247) to make sure it is compatible with all global Kaminari config choices. How far do we want to go here? At least, I'd hard code the number of items per page here.